tests/Aspire.Dashboard.Tests/Integration/StartupTests.cs

```diff
@@ -530,9 +530,9 @@ public async Task Configuration_Logging_OverrideDefaults()
 
         var options = app.Services.GetRequiredService<IOptions<LoggerFilterOptions>>();
 
-        Assert.Single(options.Value.Rules.Where(r => r.CategoryName == null && r.LogLevel == LogLevel.Trace));
-        Assert.Single(options.Value.Rules.Where(r => r.CategoryName == "Grpc" && r.LogLevel == LogLevel.Trace));
-        Assert.Single(options.Value.Rules.Where(r => r.CategoryName == "Microsoft.Hosting.Lifetime" && r.LogLevel == LogLevel.Trace));
+        Assert.Single(options.Value.Rules, r => r.CategoryName is null && r.LogLevel is LogLevel.Trace);
```

Comment:
Looks like you may have done a find-replace of "==" with "is".
AFAIK, it is an established convention throughout the dotnet repos to use "is" when checking for null and in pattern matching. However, it is not used for any other type of comparison.